modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPythonConnexionServerCodegen.java

```diff
@@ -680,14 +680,14 @@ public void postProcessParameter(CodegenParameter parameter) {
                 if (operation.bodyParam != null) {
                     Map<String, String> consume = new HashMap<>();
                     consume.put(MEDIA_TYPE, "application/json");
-                    operation.vendorExtensions.put("x-prefered-consume", consume);
+                    operation.vendorExtensions.put("x-preferred-consume", consume);
                     skipTests.put("reason", "*/* not supported by Connexion. Use application/json instead. See https://github.com/zalando/connexion/pull/760");
                 }
             }
             // Choose to consume 'application/json' if available, else choose the last one.
             if (operation.produces != null) {
                 for (Map<String, String> produce : operation.produces) {
-                    operation.vendorExtensions.put("x-prefered-produce", produce);
+                    operation.vendorExtensions.put("x-preferred-produce", produce);
```

Comment:
here in python tpyo is only in java code, mustache files already have correct spelling, there were no changes in samples, but might result in more complex schemas